Doc/library/xmlrpc.client.rst

```diff
@@ -34,10 +34,7 @@ between conformable Python objects and XML on the wire.
 
 .. class:: ServerProxy(uri, transport=None, encoding=None, verbose=False, \
                        allow_none=False, use_datetime=False, \
-                       use_builtin_types=False, *, context=None)
-
-   .. versionchanged:: 3.3
-      The *use_builtin_types* flag was added.
+                       use_builtin_types=False, headers=None, *, context=None)
```

Comment:
Does `headers` need to be a positional argument here? It seems more like `context` than like the other positional arguments to me.

Reply:
context is used only for https while headers is always used.